containerized.mk

```diff
@@ -55,3 +56,20 @@ run: ensure_image_exists
 		&& DOCKER_RUN_COMMAND+=" $$DOCKER_SWARMKIT_DOCKER_RUN_FLAGS $$DOCKER_SWARMKIT_EXTRA_RUN_FLAGS ${IMAGE_NAME} $$DOCKER_SWARMKIT_DOCKER_RUN_CMD" \
 		&& echo $$DOCKER_RUN_COMMAND \
 		&& eval $$DOCKER_RUN_COMMAND
+
+# use buildx/bake for the following targets
+.PHONY: fmt-proto
+fmt-proto:
+	$(BUILDX_CMD) bake fmt-proto
+
+.PHONY: lint
+lint:
+	$(BUILDX_CMD) bake lint
+
+.PHONY: validate-vendor
+validate-vendor:
+	$(BUILDX_CMD) bake vendor-validate
```

Comment:
Perhaps the bake and make target should have the same name (it's a bit confusing that they're named different)